src/config/inputs.rs

```diff
@@ -176,6 +178,21 @@ pub(crate) struct FleetLockInput {
     pub(crate) base_url: String,
 }
 
+/// Config for "weekly" strategy.
```

Comment:
Sidenote: I find the naming for this strategy a little confusing. "weekly" makes it seem like all it can do is do updates every week. When really, it also implements daily or every X hours, etc, right? I don't really have a better suggestion though... maybe "windowed"?

Reply:
I do not disagree :)

In the first iteration here I used "periodic", but I realized it was too generic (and I'm still considering if I can have another strategy based on https://www.freedesktop.org/software/systemd/man/systemd.time.html# later on).

Locksmith used to just call this "reboot_window" (but it allowed a single one).

If we can find a single word which conveys both the "timespans on a calendar" and "recurring every week" I'd be very happy.

Reply:
`interval`?

Reply:
what was wrong with `periodic`? One way to handle adding another one later on is to use a version maybe

Reply:
@dustymabe nothing per se, it was just feeling more generic/broad than `weekly`. But if the latter is deceiving and the former seems ok, I can revert back.

(no need to worry for future strategy names here, I just wanted to note that this may not be the only time-based one)

Reply:
Will throw another entry in the hat: `weekly_schedule`. It's more wordy, but it makes it more clear that it's about setting up a schedule tied to a week rather than a single window which happens every week.

Reply:
Ok, it looks like we have full circle without a clear winner. It seems that much confusion came from my late decision to step away from the initial pick. I have reverted everything back to use `periodic` as in the original form. How does it look like?